content/pages/meetup/2022/10-nycu.rst

```diff
@@ -0,0 +1,104 @@
+==============================================
+Sciwork Meetup October 2022 in NYCU
+==============================================
+
+:date: 2022-09-13 18:00
+:url: meetup/2022/10-nycu
+:save_as: meetup/2022/10-nycu.html
+
+Open, Free, and Balanced Software Career: a Panel Discussion
```

Comment:
Need a different title

Reply:
Yes... But I have not come up with a suitable tile. Do you have any ideas?

Reply:
What about repeating the HTML title in the same way as other sprints, e.g., https://sciwork.dev/sprint/2022/09-hsinchu ?

Reply:
Ok~ That is quite clear.